src/foam/lib/json/Outputter.java

```diff
@@ -96,19 +97,38 @@ protected void outputNull() {
   }
 
   public void outputString(String s) {
-    writer_.append("\"");
-    writer_.append(escape(s));
-    writer_.append("\"");
+    if ( multiLineOutput_ && s.indexOf('\n') >= 0 ) {
+      writer_.append("\n  ");
+      writer_.append("\"");
```

Comment:
This is really confusing to read, you can combine these append statements

instead of:
```
append("\"");
append("\"\"\n   ");
```
why not just
```
append("\"\"\"\n   ");
```